.rtx.toml

```diff
@@ -0,0 +1,4 @@
+[tools]
+yarn = "1"
+postgres = "13.12"
```

Comment:
Cool that we are specifying the version, 11 or higher was not true - one of the tests (and so, the related functionality) was failing on 11.

Reply:
Not sure how much this matters, but DEV specify it's using `13.11`

Reply:
I based this version off of whatever I had locally resolved for `postgres:13` docker container image, as pulled by our docker-compose stack, so it seems we run fine on 13.12. I can't think of a recent time a minor version upgrade broke "userspace" when it comes to Postgres - they're pretty good about retaining backwards-compatibility within a major release series.